static/src/javascripts/projects/common/modules/commercial/dfp.js

```diff
@@ -157,6 +157,14 @@ define([
             googletag.display(_.keys(slots).shift());
             displayed = true;
         },
+        displayLazyAds = function () {
+            googletag.pubads().enableSingleRequest();
```

Comment:
If you `disableSingleRequest` instead, each ad will be loaded when its `display()` is called, which avoids having to make a `refresh` call.